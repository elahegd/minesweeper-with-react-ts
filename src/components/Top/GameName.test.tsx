import React from 'react';
import { render } from "@testing-library/react";

import {GameName} from "./GameName";

describe("GameName renders coreectly", () => {
    it("get snapshot from GameName component", () => {
        const { asFragment } = render(<GameName>Minesweeper</GameName>);

        expect(asFragment()).toMatchSnapshot();
    })
})