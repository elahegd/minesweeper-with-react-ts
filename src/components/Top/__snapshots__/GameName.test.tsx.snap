// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameName renders coreectly get snapshot from GameName component 1`] = `
<DocumentFragment>
  <h1
    class="css-18kvaxm"
  >
    Minesweeper
  </h1>
</DocumentFragment>
`;
